build.sbt

```diff
@@ -187,3 +187,14 @@ lazy val native = project.in(file("native"))
     )
   )
   .enablePlugins(ScalaNativePlugin)
+
+lazy val bench = project.in(file("bench"))
```

Comment:
I'd prefer `benchmarks`, but I'm not sure the convention. I've seen a lot of Scala projects use `benchmarks`.

Reply:
(grepping the community build config file for `bench` shows a lot of variation; `benchmark` and `benchmarks` do show up, but `bench` is common)